MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentReviewModel.swift

```diff
@@ -40,48 +37,22 @@ public class PaymentReviewModel: NSObject {
     private var merchantSDK: GiniMerchant
     private var selectedPaymentProvider: PaymentProvider?
 
-    private var cellViewModels: [PageCollectionCellViewModel] = [PageCollectionCellViewModel]() {
-        didSet {
-            self.reloadCollectionViewClosure?()
-        }
-    }
-
-    var numberOfCells: Int {
-        return cellViewModels.count
-    }
-
     var isLoading: Bool = false {
         didSet {
             self.updateLoadingStatus?()
         }
     }
-    
-    var isImagesLoading: Bool = false {
-        didSet {
-            self.updateImagesLoadingStatus?()
-        }
-    }
 
     var paymentComponentsController: PaymentComponentsController
 
-    public init(with giniMerchant: GiniMerchant, document: Document?, extractions: [Extraction]?, paymentInfo: PaymentInfo?, selectedPaymentProvider: PaymentProvider?, paymentComponentsController: PaymentComponentsController) {
```

Comment:
Clarification: You removed the PaymentReview with document logic, but we will merge both implementations in the next phase. I think we are supposed to keep them both and use them based on a small configuration, or on the SDK